### PR TITLE
Make builder runnable with current nx / angular version.

### PR DESCRIPTION
**Pull request by jhwegener**
I've had some problems, to run the Builder in a NX Workspace with following versions:
"@angular-devkit/build-angular": "~12.0.4",
"gherkin-testcafe": "5.1.0",
"testcafe": "1.16.0"

I've fixed it while changing following:
* Use lodash instead of lodash-es
* rename "builder.json" to "builders.json" since this is requestet by angular
* fix path to implementation in builders.json
* add builders.json & schema.json to npm package
* dependency updates

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,22 +30,29 @@
...
-    "@typescript-eslint/eslint-plugin": "4.32.0",
-    "@typescript-eslint/parser": "4.32.0",
+    "@typescript-eslint/eslint-plugin": "4.28.4",
+    "@typescript-eslint/parser": "4.28.4",
     "dotenv": "10.0.0",
     "eslint": "7.32.0",
     "eslint-config-prettier": "8.3.0",
+    "gherkin-testcafe": "5.1.0",
     "jest": "27.2.2",
+    "lodash": "^4.17.21",
```
Comment (AdaskoTheBeAsT) [inline package.json:53]: can it be change to https://www.npmjs.com/package/lodash.ismatch? as I use only that function maybe it is not necessary to drag whole lodash :)

Comment (AdaskoTheBeAsT) [reply package.json:53]: nah - those are dev depndencies - thanks a lot for great work :)

**Review by AdaskoTheBeAsT — APPROVED** [review]

**Comment by AdaskoTheBeAsT** [comment]
Thanks a lot for contribution - after small tweak published to npm - please check

**Comment by jhwegener** [comment]
Yes, 0.9.2 is working for me. Thx for the quick merge!

**Pull request merged**